web/src/routes/TagDetails/TagDetailsArchSelect.tsx

```diff
@@ -23,7 +23,7 @@ export default function ArchSelect(props: ArchSelectProps) {
   if (!props.render) return null;
 
   return (
-    <Flex>
+    <Flex style={props.style ? props.style : undefined}>
```

Comment:
do we need the conditional operator here? The [ArchSelect](https://github.com/quay/quay/pull/2629/files#diff-ebca11beb5227e987d029367db303de98441938d334f2001261a37f85f18cd7fR121) element seems to be rendered from only one place and it has a style property.

Reply:
For re-usability of the arch selector (which we might want to use in different places in the future) I kept this optional.